ome/ome-files-cpp
# Move custom sidebars under templates

See also https://github.com/ome/ome-model/pull/36

This migration should make our Sphinx documentation 1.6.x compatible while preserving the backwards compatibility with the current version used for deployment.

To test this PR:
- review the content generated by the daily job at http://www.openmicroscopy.org/site/support/ome-files-cpp-staging/
- (optionally) build the documentation from this branch with Sphinx 1.6 eg. ``source ~/venvs/Sphinx1.6/bin/activate && cd docs/sphinx && make clean html`` and check that it compiles
